src/status_im/navigation/screens.cljs

```diff
@@ -964,7 +965,19 @@
    {:name      :screen/keycard.create.ready-to-add
     :metrics   {:track? true}
     :options   {:insets {:top? true :bottom? true}}
-    :component keycard.create/ready-to-add}])
+    :component keycard.create/ready-to-add}
+
+   {:name      :screen/keycard.factory-reset.success
+    :metrics   {:track? true}
```

Comment:
Do we have a list of screens we want to track? cc @ilmotta

Also it seems all screens in this file have `track?` to be true. Maybe we can just remove this option then and track all screens by default :thinking:

Reply:
@Parveshdhull, we don't have a list of screens to track. We basically want to track most screens, except when there's a special privacy concern even for a user who opted in. Tracking most screens allows us to understand where users are dropping off, for example. This Notion doc is supposed to be our source of truth about which events we publish https://www.notion.so/Event-Tracking-Specification-Mobile-11a8f96fb65c8032a05efcebea3a1ce6

I think we decided to use an explicit `:track?` flag because it felt a bit more safe, but we can do the opposite and have a default that always tracks (when opt-in is enabled) and then use a negative flag like `:do-not-track?` to reverse the card.

Maybe @seanstrom remember something else?